molecule/verifier/testinfra.py

```diff
@@ -55,7 +55,8 @@ def execute(self):
             self._molecule.verifier_options)
         testinfra_options['env'] = ansible.env
         testinfra_options['debug'] = self._molecule.args.get('debug', False)
-        testinfra_options['sudo'] = self._molecule.args.get('sudo', False)
+        if self._molecule.args.get('sudo'):
+            testinfra_options['sudo'] = True
```

Comment:
sudo in the context of `self._molecule.args` would apply here.

One could override testinfra to always set sudo, or simply supply on the command line.